common/live-streaming.md

```diff
@@ -84,3 +85,25 @@ For custom apps, you can enable the live streaming destination manually:
 1. Ensure that you have roles for the broadcaster (who can publish their audio/video) and the viewer (who cannot publish audio/video)
 
 ### SDK integration
+
+Use the 100ms client-side SDKs to integrate streaming in your application. See code snippets for the web SDK [here](./../features/hls).
+
+### HLS playback for the audience
+
+![HLS playback](/docs/docs/v2/hls-playback.png)
+
+You can serve HLS streaming video from a [CDN](https://www.stackpath.com/edge-academy/what-is-a-cdn/) for live playback for the audience.
+
+1. Once you start the HLS steaming, you will get an HLS URL (m3u8 URL) which you can use for playback.
+2. If you are planning to use 100ms [client-side SDKs](./../features/hls) for HLS playback, you can get the HLS URL by checking the current room state using the client SDK. Check [here](./../features/hls#current-room-status) to find out how.
+3. If you wish to use a custom integration for HLS playback (not 100ms client SDK), you can follow one of the below approaches to get the HLS URL:
```

Comment:
![Screenshot 2022-12-16 at 8 57 04 AM](https://user-images.githubusercontent.com/284612/208015252-52a91aea-1e33-4518-8e03-45ede27697e8.png)

this section is not easily readable. change this to help the user in 2 parts
1. first, give them information to decide which path to take. "why" should they choose client SDK vs server side
2. then, tell them "how" to do what's required

currently this section mixes both and is hard to grok

Reply:
done

Reply:
is this done? not seeing a clear split between "why" and "how" as i've described above in the [latest preview link](https://100ms-docs-f8t4gwsxb-100mslive.vercel.app/docs/javascript/v2/foundation/live-streaming#live-stream-playback)

Reply:
added it.